docs/src/components/navbar.tsx

```diff
@@ -31,9 +31,11 @@ export function Navbar({
         ([category, entries]) =>
           [
             category,
-            entries.filter(({ name }) =>
-              name.toLowerCase().includes(lowerCaseQuery),
-            ),
+            category.toLowerCase().includes(lowerCaseQuery)
```

Comment:
Not sure but `startsWith` might be better for this case.

Reply:
I agree with this observation, I think startsWith will make more sense here.

Reply:
Done